Description: Disble tests dependent of the trawld DBus Interface
Author: Soumya Ranjan Patnaik <soumyaranjan1812@gmail.com>
Last-Update: 2024-11-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trawldb/tests/integration_tests.rs
===================================================================
--- trawldb.orig/tests/integration_tests.rs
+++ trawldb/tests/integration_tests.rs
@@ -9,6 +9,7 @@ use trawldb::Client;
 use common::*;
 
 #[tokio::test]
+#[ignore]
 async fn filename() {
     let expected = arg_filename_default().await;
     let curr_resources = get_resources_prop().await.unwrap();
@@ -17,6 +18,7 @@ async fn filename() {
 }
 
 #[tokio::test]
+#[ignore]
 async fn merge() {
     let expected = arg_merge().await;
     let curr_resources = get_resources_prop().await.unwrap();
@@ -25,6 +27,7 @@ async fn merge() {
 }
 
 #[tokio::test]
+#[ignore]
 async fn load() {
     let expected = arg_load().await;
     let curr_resources = get_resources_prop().await.unwrap();
@@ -34,6 +37,7 @@ async fn load() {
 }
 
 #[tokio::test]
+#[ignore]
 async fn query() {
     arg_filename_default().await;
     let (q, expected) = arg_query().await;
@@ -44,6 +48,7 @@ async fn query() {
 }
 
 #[tokio::test]
+#[ignore]
 async fn query_all() {
     arg_filename_default().await;
     let expected = query_all_result().await;
@@ -54,6 +59,7 @@ async fn query_all() {
 }
 
 #[tokio::test]
+#[ignore]
 async fn edit() {
     arg_filename_default().await;
     let client = Client::new().await.unwrap();
@@ -67,6 +73,7 @@ async fn edit() {
 }
 
 #[tokio::test]
+#[ignore]
 async fn clear() {
     arg_filename_default().await;
     clear_resources().await;
